.jenkins/infrastructure/docker/build_mystikos_docker_images.Jenkinsfile

```diff
@@ -0,0 +1,139 @@
+def containers_reqs = [
+    'libcxx': ['dir': 'tests/libcxx', 'make': 'libcxx-tests'],
+    'libcxx2': ['dir': 'tests/libcxx2', 'make': 'libcxx-tests']
+]
+
+def containers_to_build = [
```

Comment:
@asvrada @vtikoo Here are the containers/tags being uploaded to Azure. Please let me know which ones are missing, which aren't needed, and which tags are incorrect.

Thanks!

Reply:
I'm wondering when you are going to invoke this Jenkins file. I am familiar with the following 4 docker images published to docker hub:

- mystikos/pytorch-test:v1.10.0
- mystikos/azure-python-sdk-storage-blob:azure-storage-blob_12.8.1
- mystikos/azure-python-sdk-keyvault-identity:azure-mgmt-keyvault_9.1.0
- mystikos/azure-python-sdk-storage:azure-mgmt-storage_19.0.0

They were built and pushed manually only for once. There are not rebuilt every time the test pipeline runs. The test scripts simply pull the pre-built, immutable image. Are we going to do the same in Azure hub?

Reply:
Yes, the same things are going to happen with Azure. This Jenkins file will only run when we need to update the containers, which I assume isn't too often.

Would you know where the dockerfiles are located for the 4 docker images you mentioned?

Reply:
Got it.

The three "azure-python-sdk-xxx" docker containers are built using this dockerfile: `solutions/python_azure_sdk/Dockerfile`

The build flags used for building these three images are mentioned in this README file:
`solutions/python_azure_sdk/README.md`.

The pytorch-test image was built and pushed to Docker Hub by running `make image && make push` under the `solutions/pytorch_test` folder. You can try the following command for `docker build`:

```bash
DOCKER_BUILDKIT=1 docker build -f Dockerfile --progress=auto --target test -t $(DOCKER_REGISTRY)/$(DOCKER_ORG)/pytorch-test:v1.10.0 --build-arg BASE_IMAGE=python:3.9-buster --build-arg PYTORCH_VERSION=v1.10.0 .
```

You can change the `${DOCKER_REGISTRY}/${DOCKER_ORG}` to the appropriate values for Azure hub.